Snakefile

```diff
@@ -1,15 +1,16 @@
-from config.ScibConfig import *
+from ScibConfig import *
 
 #configfile: "config.yaml"
 cfg = ParsedConfig(config)
 wildcard_constraints:
-    hvg = "hvg|full_feature"
+    hvg="hvg|full_feature",
+    scaling="scaled|unscaled",
+    method="[^/]+"
```

Comment:
what does the method statement mean here? Is this a regexp about the first letter not being a `/`?

Reply:
Yes, I had issues with `snakemake` not finding `method=R/seurat`

Reply:
I guess that's an issue with the config file then, no? Not really sth we have hardcoded like this somehwere?

Reply:
We save R method output in a separate `R/` directory, which, when given the path, leads to ambiguity whether to use `method=seurat` from the R integration & conversion rules or `methd=R/seurat` from the python integration rule

Reply:
Ah, I see. Okay, thanks!